meta/runtime.yml

```
@@ -472,3 +472,46 @@ plugin_routing:
       tombstone:
         removal_version: 1.0.0
         warning_text: The logicmonitor_facts module is no longer maintained and the API used has been disabled in 2017.
+    infini_export:
+      redirect: infinidat.infinibox.infini_export
+      tombstone:
```

Same for all other entries:
```suggestion
      deprecated:
```
(Sorry that I didn't notice this earlier.)